### ISSUE TITLE
Fully customizable addition and modification of lore

### ISSUE THREAD

**Issue by SuperBeacon** [Wolfieheart/ArmorStandEditor#607]
### Is your feature request related to a problem?

reason: I hope the layout style of the overall lore of the server can be unified 

### Describe the solution you'd like.

Version: Armorstandeditor-1.18.2-34.2
Requirement:
1. Support the modification and addition of GUI items in the menu, as well as the modification and addition of UI items in the menu layout
example:
#icons
zaxis:
msg: Z Axis
description:
msg: 
- "Rotate body parts along the Z Axis"
- " "
- " "
2. ToolLore hopes to support adding multiple lines of Lore
example:
requireToolLore: true
toolLore: 
- " "
- " "
- " "
- " "
- " "

### Describe alternatives you've considered.

None

### Agreements

- [X] I have searched for and ensured there isn't already an open issue regarding this.
- [X] I have ensured the feature I'm requesting isn't already in the latest supported Paper build.

**Comment by Wolfieheart** [comment]
Multi Line Tool Lore is already available via ArmorStandEditor 1.20.1-43 

The GUI configuration so the users can do it is on the backlog but will use this issue to track it going forward.

**Comment by SuperBeacon** [comment]
My server is version 1.18, can this version be used？

**Comment by Wolfieheart** [comment]
Yes everything is backwards compatible. Just read the change logs  to
double check in case you need to redo any permissions etc.

**Comment by SuperBeacon** [comment]
By the way, I think it should be allowed to add new buttons to the menu in the gui, and also to modify the position of existing buttons in the menu

**Comment by SuperBeacon** [comment]
> 1. example:
>    #icons
>    zaxis:
>    msg: Z Axis
>    description:
>    msg:
> 
> * 'Rotate body parts along the Z Axis'
> * ' '
> * ' '

example:
#icons
slot: 0
zaxis:
msg: Z Axis
description:
msg:
'Rotate body parts along the Z Axis'
' '
' '
